### DISCUSSION TITLE
Major Update to Synapse_CoR

### DISCUSSION THREAD

**Discussion by ProfSynapse** [ProfSynapse/Professor-Synapse#17] [Announcements]
Hey chatter just wanted to share I've posted a fairly significant update to Synapse_CoR. As of yesterday it seems like OpenAI made some changes and I've noticed that the functionality became too verbose and got in a loop of just spinning up a new agent in every output and didn't really move forward with the goals.

I also wanted to give it more context around what tools it might have access to in ChatGPT+, especially since they are going to pull everything together under one chat to access everything at once.

Test it out and let me know what feedback you have!

**Comment by thvi** [comment]
Thank you! Is it possible to optimize it by integrating the rule "Answer in {language}" to get systematic answers in the language spoken by users whose language is not English and keeping the word limit at 1500 words?

**Reply by thvi** [reply]
Ok, maybe I could remove this rule "-use emojis freely to express yourself" and replace it with "-reply in FR".

**Comment by Ethiventure** [comment]
Do you know what ChatGPT changed? I'm using my own variations and I'm wondering what to update. BTW I had one day when my agents could proceed to a proper next response without me but that's rare now, it coincided with lots of ChatGPT changes. I'm using instructions like /wua=ProceedAutonomouslyToNextStepWithoutUserApproval ![image](https://github.com/ProfSynapse/Synapse_CoR/assets/130914871/dfa8530a-0f29-41ad-ba75-792ecc9abe17) ![image](https://github.com/ProfSynapse/Synapse_CoR/assets/130914871/54e02664-c43b-417e-bef2-457ba7bf90bf)

**Reply by ProfSynapse** [reply]
I can't be sure but they will likely announce a lot at their dev conference.

**Comment by catsncode** [comment]
It would be great if there was a way to change "modes" from a more verbose
like the default to a quicker response for easier questions that do not
need agents, etc.  Thanks!

**Comment by davecohoe** [comment]
I've tried a slimmed down version GPTprompt.txt, and the custominstruction.txt, both in GPT3.5 custom instructions. It does seem that OpenAI changed something, because it seems more difficult to init a subject expert, Prof Synapse seems much more verbose, and waffles on about discussion, when you want it to just do the thing you asked. I also found that it will bounce back from an expert to Prof Synapse, in effect, starting the conversation over, without completing my task.
